dwa_planner:
  ros__parameters:
    max_speed: 0.15
    max_turn: 2.5
    step_time: 0.1
    goal_tolerance: 0.05
    safety_margin: 0.3
    sim_steps: 100
    num_samples: 10000
    goal_weight: 5.0
    heading_weight: 2.0
    collision_penalty: 100000.0
    smoothness_weight: 0.1
